### PR TITLE
Ca 23 get /kingdom/resources

### PR DESCRIPTION
**Pull request by ZaitzeV16**
New endpoint to KingdomController.

### CODE REVIEW

**Comment by ZaitzeV16** [comment]
Test still needed.

File: src/main/java/com/greenfoxacademy/chamaaragonite/exceptions/VeryMuchMissingResourceException.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.greenfoxacademy.chamaaragonite.exceptions;
+
+public class VeryMuchMissingResourceException extends RuntimeException {
+
+  public VeryMuchMissingResourceException() {
+    super("A very funny exception occurred, happy Easter!");
```
Comment (csadesz) [inline src/main/java/com/greenfoxacademy/chamaaragonite/exceptions/VeryMuchMissingResourceException.java]: 🥚

File: src/main/java/com/greenfoxacademy/chamaaragonite/dtos/ResourceResponseDTO.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import com.greenfoxacademy.chamaaragonite.resource.Resource;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
```
Comment (csadesz) [inline src/main/java/com/greenfoxacademy/chamaaragonite/dtos/ResourceResponseDTO.java]: Do we need this?

File: src/main/java/com/greenfoxacademy/chamaaragonite/kingdom/KingdomService.java [hunk 3]
```diff
@@ -10,4 +10,6 @@
   public Kingdom generateKingdom(String kingdomName);
 
   public Kingdom updateKingdom(Kingdom kingdom);
+
+  public Kingdom findByAppUserUsername(String appUserUsername);
```
Comment (csadesz) [inline src/main/java/com/greenfoxacademy/chamaaragonite/kingdom/KingdomService.java]: access modifiers :(

File: src/main/java/com/greenfoxacademy/chamaaragonite/dtos/ResourceResponseDTO.java (hunk 2 above)
Comment (ZaitzeV16) [reply src/main/java/com/greenfoxacademy/chamaaragonite/dtos/ResourceResponseDTO.java]: yes

File: src/main/java/com/greenfoxacademy/chamaaragonite/kingdom/KingdomService.java (hunk 3 above)
Comment (ZaitzeV16) [reply src/main/java/com/greenfoxacademy/chamaaragonite/kingdom/KingdomService.java]: yes

**Pull request merged**